lightning-block-sync/src/poll.rs

```diff
@@ -140,6 +141,19 @@ impl ValidatedBlockHeader {
 
 		Ok(())
 	}
+
+    /// Returns the [`BestBlock`] corresponding to this validated block header,
+    /// which can be passed into [`ChannelManager::new`] as part of its
+    /// [`ChainParameters`]. Useful for ensuring that the [`SpvClient`] and
+    /// [`ChannelManager`] are synced to the same block during a fresh start.
```

Comment:
nit: s/synced/initialized

(Technically, `ChannelManager` is being synced by `SpvClient`.)